.gitmodules

```diff
@@ -1,16 +1,16 @@
 [submodule "backpack-android"]
 	path = backpack-android
 	url = https://github.com/Skyscanner/backpack-android
-	##ignore = all
+	###ignore = all
 [submodule "backpack-ios"]
 	path = backpack-ios
 	url = https://github.com/Skyscanner/backpack-ios
-	##ignore = all
+	###ignore = all
 [submodule "backpack-react-native"]
 	path = backpack-react-native
 	url = https://github.com/Skyscanner/backpack-react-native
-	##ignore = all
+	###ignore = all
 [submodule "backpack"]
 	path = backpack
 	url = https://github.com/Skyscanner/backpack
-	##ignore = all
+	###ignore = all
```

Comment:
This shouldn't happen but on the other hand it's not affecting anything so oh well